src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -455,6 +460,17 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
      * @return {void}
      */
     fetchFeedItemsSuccessCallback = (feedItems: FeedItems): void => {
+        const { logger } = this.props;
+
+        // Only emit metric if has >1 activity feed items (there should always at least be the current version)
+        if (feedItems.length > 1) {
+            mark(MARK_NAME_DATA_READY);
```

Comment:
Should we always set the mark, but only log if the feedItems array is greater than one?